hw/drivers/sensors/bma253/syscfg.yml

```diff
@@ -47,7 +47,7 @@ syscfg.defs:
         value: -1
     BMA253_LOG:
         description: 'Enable BMA253 logging'
-        value: 0
+        value: 1
```

Comment:
Disable logging by default, should be enabled only if needed

Reply:
will do